openbas-front/src/components/fields/SelectField.jsx

```diff
@@ -40,9 +43,9 @@ const SelectField = (props) => {
           </MUISelect>
         )}
       />
-      {helperText && (
-        <FormHelperText variant={others.variant}>
-          {helperText}
+      {!!error && (
+        <FormHelperText variant={others.variant} error={!!error}>
+          {helperText ?? error?.message}
```

Comment:
I think helperText can be removed
Is no longer in the params of the component and it's handle by zod with the error.message

Reply:
HelperText needs to be mentionned, else no text is displayed.